### ISSUE TITLE
Corresponds to authorization code grant flow.

### ISSUE THREAD

**Issue by SawamiWataru** [personium/personium-core#213]

**Comment by SawamiWataru** [comment]
Corresponded part of code grant flow.
Currently __authz endpoints can return "Code" only from cookie information. (ID - Password is not supported yet)

To obtain cookies, you need to do the following on the __token endpoint:
```
POST
__token
grant_type=password&username={id}&password={password}&p_cookie=true
```

If cookie is set, it is possible to return code from the __authz endpoint.
```
GET
__authz?response_type=code&redirect_uri={redirectUri}&client_id={appUrl}&state={state}
```

**Comment by SawamiWataru** [comment]
Released in v1.6.9.
